ccdproc/tests/test_ccdmask.py

```diff
@@ -0,0 +1,222 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# This module implements the base CCDData class.
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from numpy.testing import assert_array_equal
+
+import numpy as np
+
+from astropy.tests.helper import pytest
+
+from ..core import ccdmask
+from ..ccddata import CCDData
+
+
+def test_ccdmask_no_ccddata():
+    # Fails when a simple list is given.
+    with pytest.raises(ValueError):
+        ccdmask([[0, 0, 0], [0, 0, 0], [0, 0, 0]])
+
+
+def test_ccdmask_not_2d():
+    # Fails when a CCDData has less than 2 dimensions
+    with pytest.raises(ValueError):
+        ccdmask(CCDData(np.ones(3), unit='adu'))
+
+    # Fails when scalar
+    with pytest.raises(ValueError):
+        ccdmask(CCDData(np.array(10), unit='adu'))
+
+    # Fails when more than 2d
+    with pytest.raises(ValueError):
+        ccdmask(CCDData(np.ones((3, 3, 3)), unit='adu'))
+
+
+def test_ccdmask_pixels():
+    flat1 = CCDData(np.array([[
```

Comment:
[repost] 
Could a comment be added here about the intent of the test arrays? Are they snippets from larger images, should they have bad pixels or are those only added in the body of the test, etc.

Doesn't need to be super long, just enough so it is clearer what can (and cannot) be modified in the future.

@joshwalawender can maybe address this most easily.